Neuron/Models/Realm/WalletModel.swift

```diff
@@ -9,10 +9,29 @@
 import UIKit
 import RealmSwift
 
+enum WalleIconType: String {
+    case dog = "icon_wallet_dog"
+    case fish = "icon_wallet_fish"
+    case owl = "icon_wallet_owl"
+    case parrot = "icon_wallet_parrot"
+    case rat = "icon_wallet_rat"
+    case squirrel = "icon_wallet_squirrel"
+    case fox = "icon_wallet_fox"
+    case tiger = "icon_wallet_tiger"
+
+    var image: UIImage {
+        return UIImage(named: rawValue)!
+    }
+
+    static var allType: [WalleIconType] {
```

Comment:
Prefer `CaseIterable` and `allCases`. See https://github.com/cryptape/neuron-ios/blob/7fd9ee2f6631d604151a9ef4a105871413ba7738/Neuron/Services/Ethereum/EthereumNetwork.swift#L53 for an example.